src/site/content/en/blog/app-shortcuts/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Get things done quickly with app shortcuts
+subhead: App shortcuts give quick access to a handful of common actions that users need frequently.
+authors:
+  - beaufortfrancois
+  - jungkees
+date: 2020-05-27
+updated: 2020-05-27
+hero: hero.jpg
+thumbnail: thumbnail.jpg
+alt: A photo of an Android phone showing an app shortcuts menu
+description: App shortcuts give quick access to a handful of common actions that users need frequently.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+  - fugu
+  - progressive-web-apps
+---
+
+To improve users' productivity and facilitate re-engagement with key tasks, the
+web platform now supports app shortcuts. They allow web developers to provide
+quick access to a handful of common actions that users need frequently.
```

Comment:
```suggestion
To improve users' productivity and facilitate re-engagement with key tasks,
Chromium-based browsers now supports app shortcuts. They allow web developers to provide
quick access to a handful of common actions that users need frequently.
```
Sorry but we can't say this unless Firefox and Safari also support it. If they do then you can disregard this comment. We have specifically had complaints about us implying that Chromium-based browsers are the web platform.

Reply:
It would also be nice to link to an MDN browser compatibility table

Reply:
Because manifest are part of the manifest spec, which is supported on all browsers I think it's ok in this case to say this. We're not introducing a brand new API, it's an addition to the manifest spec that is widely deployed.

Note that Chromium-based browsers isn't very user friendly.

Reply:
For info, my original writing before copy-editing was "app shortcuts are added to the web platform". It was changed to "the web platform now supports app shortcuts" as using active voice is a best practice.

Reply:
Pinged you and @petele on Chat to resolve.

Reply:
Noting for the record that I expressed my concerns to Pete and Francois over Chat but they still feel that "web platform" is an appropriate phrase here so we're keeping it.